apps/spinner/src/App.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from "zarkit/react";
+import "./App.scss";
+import Spinner from "./spinner"
+import ChanceWheel from "./chance-wheel"
+
+
+function App(){
+
+const [state, setState] = useState({
+value: 6 , 
+isSpinner:false
+})
+
+//router
+function routeChange(){
```

Comment:
naming a toggle function `routeChange` is not good idea
you can use something like `toggleSpinner` instead

also for implement this if you use separate variables for states you can use:
```
setIsSpinner(!isSpinner);
```
but even if you still want to use object for your state still could be done with:
```
let temp = {...state};
temp.isSpinner = !temp.isSpinner;
setState(temp);
```
Again: naming a state `"state"` is not a good idea.